yoti_python_sdk/doc_scan/session/create/check/face_match.py

```diff
@@ -32,6 +32,9 @@ def manual_check(self):
     def to_json(self):
         return {"manual_check": self.manual_check}
 
+    def include_null_values(self):
+        return False
+
```

Comment:
```suggestion
    def to_json(self):
        return remove_null_values({
            "manual_check": self.manual_check,
        })

```

Reply:
I was thinking we're a bit clearer with our intentions if we leave them as separate, but I think my preference to not have clients exposed to the include_null_values method overrides that, so I've changed in ece2cad